src/content-helper/dashboard-widget/components/top-posts.tsx

```diff
@@ -87,12 +90,38 @@ export function TopPosts() {
 				onChange={ ( event ) => {
 					if ( isInEnum( event.target.value, Metric ) ) {
 						setMetricFilter( event.target.value as Metric );
+						setPage( 1 );
 					}
 				} }
 			/>
 		</div>
 	);
 
+	const navigation: JSX.Element = (
+		<div className="parsely-top-posts-navigation">
+			<button
+				className="parsely-top-posts-navigation-prev"
+				disabled={ page <= 1 }
+				onClick={ () => setPage( page - 1 ) }
+			>
+				{ '<< ' + __( 'Previous', 'wp-parsely' ) }
```

Comment:
Wondering whether we should include the `<<` and `>>` within the localized string. I'm thinking of RTL languages that might place them in a different position within the string.

You can also try the HTML entities `&laquo;`/`&raquo;` instead of `<<`/`>>` just to see if they look better, and keep whatever looks better to you.

Reply:
Moved the `<<` and `>>` to the translatable string. However I couldn't use the HTML entities, as they were being printed verbatim. Maybe the i18n library doesn't support those?

Reply:
Ah, you're right about the entities. React makes this complicated. Let's leave as is.